README.md

```diff
@@ -18,3 +18,12 @@ To generate and encrypt secrets for pipeline environments:
 https://github.com/inviqa/k8s-project-cluster/blob/0.1.x-dev/docs/cluster-management/sealed-secrets/README.md
 
 [Workspace]: https://github.com/my127/workspace
+
+## How to test?
+There are two modes of this harness: static (for CI environment) and dynamic (for local environment).
+You can test both of them using below command:
```

Comment:
Extra note to add:
Static mode is a fixed file system, and dynamic uses mutagen to sync file changes with the console and php-fpm containers.